webpack/globalIdHelper.js

```diff
@@ -0,0 +1,11 @@
+const idSeparator = '-';
+const versionSeparator = ':';
+const defaultVersion = '01';
+
+export const decodeId = model => {
+  const split = atob(model.id).split(idSeparator);
+  return parseInt(split[split.length - 1], 10);
+};
+
+export const encodeId = (typename, id) =>
+  btoa([defaultVersion, versionSeparator, typename, idSeparator, id].join(''));
```

Comment:
Not crucial but I suppose that this should be provided by core rather than every plugin creating its own.

Reply:
Yes, this should move to core at some point.